caas/kubernetes/clientconfig/k8s.go

```diff
@@ -40,22 +39,15 @@ func NewK8sClientConfig(
 	contextName, clusterName string,
 	credentialResolver K8sCredentialResolver,
 ) (*ClientConfig, error) {
-	if reader == nil {
```

Comment:
reader now appears unused in the new code
i suspect there will be failing tests now also

Reply:
Done & Done